### PR TITLE
cosmos grants integration

### LINKED ISSUE #10176: Cosmos Integration for Grants
**User Story:**
As a grant owner
I want to be able to create grants for Cosmos blockchain
So that I can receive contributions to my ATOM address

As a grant donor
I want to be able to checkout grants using Keplr wallet
So that I can contribute to grants on Cosmos blockchain network

**Acceptance Criteria**
GIVEN I am on the grant creation page
WHEN I click on the "Select a blockchain" dropdown
THEN I see Cosmos as an option

GIVEN I am on the grants cart page
WHEN I click on "Cosmos" tab
THEN I see Cosmos grants in my cart

GIVEN I am on the "Cosmos" tab in my cart
WHEN I click on the currency select option of a grant
THEN I can select "ATOM" in the list of tokens

GIVEN I have selected "ATOM" as currency for a grant in Cosmos tab
WHEN I click on "Payout" button
THEN I see the Keplr wallet extension prompt to sign the transaction

GIVEN I have successfully signed the transaction with Keplr wallet extension
WHEN I am redirected to the grants explorer page
THEN I see the "Thank you for contributing" modal

GIVEN I have successfully contributed to grants
WHEN The contribution is still pending on-chain
THEN Then a background task checks the on-chain status of the transaction at intervals

**Product & Design Links:**

**Tech Details:**
Crosschain Integration document: https://docs.google.com/document/d/19wVbwJYGEgkkdeNj-IiTq-PmQ3a5kBOR66Lhf-vS-xQ

**Open Questions:**

**Notes/Assumptions:**

### PR DESCRIPTION
**Pull request by chibie**
##### Description

**User Story:**
As a grant owner
I want to be able to create grants for Cosmos blockchain
So that I can receive contributions to my ATOM address

As a grant donor
I want to be able to checkout grants using Keplr wallet
So that I can contribute to grants on Cosmos blockchain network

##### Refers/Fixes

closes #10176 

##### Testing

https://share.vidyard.com/watch/ByGw4sJaMyCCm2JsuE47y8?

### CODE REVIEW

File: app/assets/v2/js/grants/cart/rsk_extension.js [hunk 1]
```diff
@@ -19,15 +19,13 @@ const contributeWithRskExtension = async(grant, vm, modal) => {
...
-      modal.closeModal();
       _alert({ message: 'Please download or enable Nifty Wallet extension' }, 'danger');
       return;
     }
 
     if (!ethereum.selectedAddress) {
-      modal.closeModal();
       return onConnect().then(() => {
-        modal.openModal();
+        console.log('wallet connected');
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/cart/rsk_extension.js:28]: did we intend on changing this? not sure why remove the modal instance

File: app/grants/sync/binance.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
+import logging
+
 import requests
 from grants.sync.helpers import record_contribution_activity
 
+logger = logging.getLogger(__name__)
```
Comment (thelostone-mc) [inline app/grants/sync/binance.py:6]: do we need this ?

Comment (chibie) [reply app/grants/sync/binance.py:6]: yea... it was already being used in L45 but not declared (bug)

File: app/assets/v2/js/grants/cart/rsk_extension.js (hunk 1 above)
Comment (chibie) [reply app/assets/v2/js/grants/cart/rsk_extension.js:28]: this was a bug as `modal` isn't passed as an input parameter in grants extension (only in bounty)

**Review by thelostone-mc — APPROVED** [review]

File: app/assets/v2/js/grants/_new.js [hunk 3]
```diff
@@ -396,7 +400,8 @@ const grant_chains = [
 
 if (document.contxt.is_staff) {
```
Comment (schultztimothy) [inline app/assets/v2/js/grants/_new.js:401]: Should Cosmos be added to the default blockchain dropdown and not just the staff one?

**Review by schultztimothy — APPROVED** [review]
I left one comment about adding cosmos to the default dropdown when creating a new grant. I was playing around with it locally and wasn't able to see it as an option.

Codewise lgtm!

File: app/assets/v2/js/grants/_new.js (hunk 3 above)
Comment (gcc20) [reply app/assets/v2/js/grants/_new.js:401]: @schultztimothy it's usually in staff until rev team gives the go ahead to make it public

Comment (chibie) [reply app/assets/v2/js/grants/_new.js:401]: @schultztimothy it's usually in staff until rev team gives the go ahead to make it public

**Pull request merged**